content/science/how-to-cite/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: How to Cite
+color: red
+class: color-footer
+---
+
+When using Open Force Field parameters or software in your research, please cite the appropriate works from below. If multiple citations are listed for an item, we recommend citing all of the suggested references.
+
+### Force fields
+
+* Parsley ([Qiu et al., 2021](https://pubs.acs.org/doi/10.1021/acs.jctc.1c00571) + [Zenodo DOIs](https://github.com/openforcefield/openff-forcefields#open-force-fields))
+
+* Sage ([Qiu et al., 2021](https://pubs.acs.org/doi/10.1021/acs.jctc.1c00571) + [Zenodo DOIs](https://github.com/openforcefield/openff-forcefields#open-force-fields))
+ 
+### Software
+
+* Toolkit ([Mobley et al., 2018](https://pubs.acs.org/doi/10.1021/acs.jctc.8b00640) + [latest version](https://zenodo.org/record/6310995#.YkdRtpPMLMI))
+
+* BespokeFit ([Zenodo DOI](https://zenodo.org/record/5172174#.YkdU1JPMLMI))
+
+* Evaluator ([Boothroyd et al. preprint](https://zenodo.org/record/5695016#.YkdTX5PMLMI) + [Zenodo DOI](https://zenodo.org/record/5695016#.YkdTX5PMLMI))
```

Comment:
In addition to these changes, the link for the Boothroyd preprint is wrong below

```suggestion
* OpenFF Toolkit ([Mobley et al., 2018](https://pubs.acs.org/doi/10.1021/acs.jctc.8b00640) + [Specific version DOI](https://zenodo.org/record/6310995#.YkdRtpPMLMI))

* OpenFF BespokeFit ([Zenodo DOI](https://zenodo.org/record/5172174#.YkdU1JPMLMI))

* Evaluator ([Boothroyd et al. preprint](https://zenodo.org/record/5695016#.YkdTX5PMLMI) + [Specific version DOI](https://zenodo.org/record/5695016#.YkdTX5PMLMI))
```